charts/retool/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: retool
 description: A Helm chart for Kubernetes
 type: application
-version: 6.0.14
+version: 6.0.13
```

Comment:
i think you can only up this to 6.0.15

Reply:
I think I can go back to 6.0.13 if I removed the release from Azure, no? 

https://retoolconfluence.atlassian.net/wiki/spaces/EPD/pages/172130364/Publishing+a+new+helm+chart+version